examples/contrib/message-generator.py

```diff
@@ -32,14 +32,14 @@
 from pymodbus.register_read_message import *
 from pymodbus.register_write_message import *
 from pymodbus.compat import IS_PYTHON3
-
+from pymodbus.pdu import ExceptionResponse
 #--------------------------------------------------------------------------#
 # initialize logging
 #--------------------------------------------------------------------------#
 import logging
 modbus_log = logging.getLogger("pymodbus")
 
-
+exp = ExceptionResponse(0x01, 0x0B)
```

Comment:
Is this being supported ? Generation of exception response?

Reply:
No, this was supposed to go away in a cleanup.